### PR TITLE
testing/libsoundio: new aport

### PR DESCRIPTION
**Pull request by Mark-Weston**

### CODE REVIEW

File: testing/libsoundio/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Maintainer: Mark Jynx <markjynx@cock.li>
+pkgname="libsoundio"
+pkgver="2.0.0"
+pkgrel=0
+pkgdesc="lightweight abstraction over various sound drivers"
+url="https://libsound.io"
+arch="all"
+options="!check"  # No test suite.
+license="MIT"
+depends="alsa-lib-dev"
```
Comment (maxice8) [inline testing/libsoundio/APKBUILD]: i doubt this should be in depends. Most likely should be in makedepends

File: testing/libsoundio/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+makedepends="cmake"
+subpackages="$pkgname-dev"
+source="${pkgname}-${pkgver}.tar.gz::https://github.com/andrewrk/libsoundio/archive/$pkgver.tar.gz"
+builddir="$srcdir/$pkgname-$pkgver"
```
Comment (maxice8) [inline testing/libsoundio/APKBUILD]: can be removed

**Comment by maxice8** [comment]
```
MC:[AL1]:testing/libsoundio/APKBUILD:14:builddir can be removed as it is the default value
MP:[AL32]:testing/libsoundio/APKBUILD:13:unnecesary usage of braces: ${pkgname}
MP:[AL32]:testing/libsoundio/APKBUILD:13:unnecesary usage of braces: ${pkgver}
TP:[AL3]:testing/libsoundio/APKBUILD:2:pkgname must not be quoted
TP:[AL4]:testing/libsoundio/APKBUILD:3:pkgver must not be quoted
```

**Comment by maxice8** [comment]
Proposed patch:

```
diff --git a/testing/libsoundio/APKBUILD b/testing/libsoundio/APKBUILD
index eae9714057..36975775a3 100644
--- a/testing/libsoundio/APKBUILD
+++ b/testing/libsoundio/APKBUILD
@@ -7,8 +7,8 @@ url="https://libsound.io"
 arch="all"
 options="!check"  # No test suite.
 license="MIT"
-depends="alsa-lib-dev"
-makedepends="cmake"
+depends_dev="alsa-lib-dev"
+makedepends="cmake $depends_dev"
 subpackages="$pkgname-dev"
 source="${pkgname}-${pkgver}.tar.gz::https://github.com/andrewrk/libsoundio/archive/$pkgver.tar.gz"
 builddir="$srcdir/$pkgname-$pkgver"
```

File: testing/libsoundio/APKBUILD (hunk 1 above)
Comment (Mark-Weston) [reply testing/libsoundio/APKBUILD]: At compile time, libsoundio checks which backends are available (ALSA, PulseAudio, JACK...) and compiles against them. So at least one of them should be there.

Perhaps change alsa-lib-dev to alsa-lib? I don't think that the headers are needed.

**Comment by Mark-Weston** [comment]
Applied.

**Comment by maxice8** [comment]
Please squash  them into a single commit

**Pull request closed without merging**